site/content/docs/main/locations.md

```diff
@@ -166,6 +171,40 @@ During backup creation:
 velero backup create full-cluster-backup
 ```
 
+### Create a storage location that uses different credentials
+
+If your object storage provider plugin supports it, it is possible to create additional `BackupStorageLocations` that use their own credentials.
```

Comment:
`If your object storage provider plugin supports it`

I'm not clear on whats needs to be supported? How would i know that my plugin supports this?

Reply:
resolving since i think you addressed this